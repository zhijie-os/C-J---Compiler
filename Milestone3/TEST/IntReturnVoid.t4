int foo()
{
    int j;
    j = 4 ;
    return;
}



main(){}
