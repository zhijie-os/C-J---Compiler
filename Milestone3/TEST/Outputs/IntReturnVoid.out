Line 5, function foo returns wrong type.
